ayup/portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,14 @@ function addRandomFact() {
   factContainer.innerHTML = newFact;
 }
 
+/**
+ * Fetch the facts from the server
+ */
+async function getRandomFactUsingAsyncAwait() {
+  const response = await fetch('/data');
+  const fact = await response.text();
```

Comment:
I don't think you need this second await. Do you?

Reply:
`response.text()` returns a promise.

https://developer.mozilla.org/en-US/docs/Web/API/Body/text